src/util/client/logentriesLogger.js

```diff
@@ -7,12 +7,19 @@ import LE from 'le_js';
 * @param {String} options.token
 * @param {String} options.level
 */
-export default function ClientLogentriesLogger({ token }) {
-  LE.init({
-    token,
-    no_format: true,
-    page_info: 'per-page'
-  });
+export default function ClientLogentriesLogger({ name, token }) {
+  try {
+    // If a LE logger does not exist with this name, this will throw
+    LE.to(name);
```

Comment:
This goes down this path: https://github.com/rapid7/le_js/blob/master/src/le.js#L342

The issue we're seeing comes from the init call being made for every logger instance (including children) when a logentries token is present. This was introduced by the refactor in #68